### PR TITLE
Add RSS icons

### PR DESCRIPTION
**Pull request by Stephanie0829**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -9,8 +9,19 @@ <h1 class="title page-title">The Teach LA Blog</h1>
   We love what we do, and we want to share it with you! Here are some insights from our members on how our club operates: how we teach, write code, and create a community!
 </p>
 <p>
-  Interested in more formal reflections on the inner workings of our club? Check out <a class="tla-link" href="{{site.baseurl}}/reports">reports section</a>. Curious in how we handle teaching an artificial intelligence and machine learning class at a high school, or our joint efforts to provide AI education? Peep the <a class="ai-link" href="https://uclaacmai.github.io/blog/" target="_blank" rel="noopener noreferrer">ACM AI blog!</a>
+  Interested in more formal reflections on the inner workings of our club? Check out 
+  <a class="tla-link" href="{{site.baseurl}}/reports">reports section</a>.
+  Curious in how we handle teaching an artificial intelligence and machine learning class at a high school, or our joint efforts to provide AI education? Peep the 
+  <a class="ai-link" href="https://uclaacmai.github.io/blog/" target="_blank" rel="noopener noreferrer">ACM AI blog</a>!
```
Comment (mattxwang) [inline blog.html:15]: Is this just a whitespace change? If so, do you mind reversing it?

File: _layouts/report.html [hunk 2]
```diff
@@ -16,6 +20,11 @@ <h1 class="title page-title">Report: {{page.title}}</h1>
     <hr class="divider" />
     <p>
         Thank you for reading this report, <b>{{page.title}}</b>. It was authored by the Teach LA community, with significant contributions from {{page.author}}.
+        </br></br>Subscribe to our RSS feed -
```
Comment (mattxwang) [inline _layouts/report.html:23]: I would avoid using `</br>` for spacing; let's instead use `margin` or `padding` if possible!

File: reports.html [hunk 3]
```diff
@@ -6,8 +6,14 @@
 <h1 class="title page-title">Reports, Reflections, & Accountability</h1>
 <p>
     As an organization, we believe it's important to constantly reflect and grow. In the spirit of one of our core values, transparency, we invite you to reflect and grow with us.
+    </br></br>Read more reports via RSS -
```
Comment (mattxwang) [inline reports.html:9]: Similar note on using `<br>` on spacing.

File: _layouts/post.html [hunk 4]
```diff
@@ -35,22 +35,34 @@ <h1 class="title page-title">{{page.title}}</h1>
...
 
 <hr class="divider" />
 <div class = "social">
 Enjoyed the read? Share on social media! <br/>
 <div class = "social-buttons">
 <a class="fb-share-button"  data-layout="button" data-size="large" target="_blank" href="" class="fb-xfbml-parse-ignore">Share</a>
 <a class="twitter-share-button" href="https://twitter.com/share?ref_src=twsrc%5Etfw" data-size="large" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
-</div></div>
+</div>
+</br>Subscribe to our RSS feed!</br>
```
Comment (mattxwang) [inline _layouts/post.html:59]: Similar note on using `<br>` for spacing.

**Review by mattxwang — CHANGES_REQUESTED** [review]
Hi Stephanie! Overall, these changes are looking great, and I'm glad you took a special note on making sure our icon links are accessible! 

I left a few general notes, but also one big idea I want to touch on: `aria-labelledby` should refer to `id`s, as they're unique, rather than classes. So, you'd need to change your hidden labels to use `id` instead of `class="icon-label"`. You may need to generate a few `id`s, since the element appears multiple times on one page. [More on that in the MDN docs.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)

One other small note: I would slightly edit the copy to use language like:

* "Read posts via RSS"
* "Read reports via RSS"

Since RSS doesn't really give you "more" content, per se!

Once we deal with those, I think we're good to merge! Thanks for all your work so far 😊

File: blog.html (hunk 1 above)
Comment (Stephanie0829) [reply blog.html:15]: Oh yeah, I apologize for that! I think I changed it personally to see the code better but forgot to change it back.

**Review by mattxwang — APPROVED** [review]
Great work Stephanie! I think this is good to merge 😊 

I'll let you do the honours, please use the squash and merge option!

**Pull request merged**